operatorconfig/driverconfig/powerscale/v2.13.0/controller.yaml

```diff
@@ -239,7 +239,7 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: csi-metadata-retriever
-          image: quay.io/dell/container-storage-modules/csi-metadata-retriever:nightly
+          image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v2.13.0
```

Comment:
this image does not exist, 1.10.0 is latest
https://quay.io/repository/dell/container-storage-modules/csi-metadata-retriever?tab=tags&tag=latest